dbt/models/iasworld/schema.yml

```diff
@@ -34,6 +43,10 @@ sources:
       - name: owndat
       - name: pardat
       - name: permit
+        freshness:
+          filter: date_format(date_parse(permdt, '%Y-%m-%d %H:%i:%s.0'), '%Y') = date_format(current_date, '%Y')
```

Comment:
I wasn't totally sure what the best field to use for this filter was, since `permit` doesn't have a `taxyr` column. I went ahead with `permdt` because so far it's consistently never null.

Reply:
I don't know enough about this table yet to add much. I think we'll just need to keep an eye on it and update it as we get more info.